UPDATING.md

```diff
@@ -24,6 +24,11 @@ assists users migrating to a new version.
 
 ## Airflow Master
 
+### Deprecation chain function
+
+Bit operation like `>>` or `<<` are recommended for setting the dependency, which is easier to explain.
+The `airflow.utlis.helpers.chain` function will be deprecated.
```

Comment:
FYI: A function is deprecated when it still exists and works but gives a warning. This is not what we did in this PR.

Reply:
@ashb , just create a minor pr(https://github.com/apache/airflow/pull/4953) to correct. Thanks for pointing out. I should review more carefully.

Reply:
@ash @feng-tao Sorry for the incorrect describe I add.